### ISSUE TITLE
Bug - Interface not load

### ISSUE THREAD

**Issue by puimpsyc** [yogeshojha/rengine#768]
Hi! , I installed everything, I access in 127.0.0.1:8000 but the interface does not load well.
![Sin título](https://user-images.githubusercontent.com/77821523/209229730-c445b1b0-d34a-4fef-a8c9-2103388f1cdd.png)

**Comment by gearcapitan** [comment]
the same problem 
![imagen](https://user-images.githubusercontent.com/13740942/209265611-966df944-9a9a-4ded-ab7c-3f5828046d51.png)

**Comment by barbixxxa** [comment]
https://github.com/yogeshojha/rengine/issues/769#issuecomment-1364272283

**Comment by paweloque** [comment]
Did you find a solution for that? I have the same problem. It looks like the collectstatic is not executed during the build.

**Comment by zinwelzl** [comment]
same problem here, nothing helps

https://rengine.wiki/install/quick/

instalattion ok, but https://127.0.0.1/ nothing

0.0.0.0:8000 just white page

what is going on?
how to repair this?


docker ps
CONTAINER ID   IMAGE                                                     COMMAND                  CREATED          STATUS                          PORTS                                       NAMES
4d0fcf8d3c99   peterdavehello/tor-socks-proxy                            "/usr/bin/tor -f /et…"   4 minutes ago    Up 4 minutes (healthy)          8853/udp, 9150/tcp                          rengine_tor_1
c672d3eadaa2   nginx:alpine                                              "/docker-entrypoint.…"   4 minutes ago    Restarting (1) 16 seconds ago                                               rengine_proxy_1
3248a8272ccc   docker.pkg.github.com/yogeshojha/rengine/rengine:latest   "/usr/src/app/entryp…"   4 minutes ago    Up 4 minutes                    0.0.0.0:8000->8000/tcp, :::8000->8000/tcp   rengine_web_1
b4d8530ce894   rengine_celery-beat                                       "/usr/src/app/beat-e…"   4 minutes ago    Up 4 minutes                                                                rengine_celery-beat_1
ca65633e8f35   rengine_celery                                            "/usr/src/app/celery…"   4 minutes ago    Up 4 minutes                                                                rengine_celery_1
92446d487abf   redis:alpine                                              "docker-entrypoint.s…"   22 minutes ago   Up 4 minutes                    6379/tcp                                    rengine_redis_1
b681fbce55dd   postgres:12.3-alpine                                      "docker-entrypoint.s…"   22 minutes ago   Up 4 minutes                    0.0.0.0:5432->5432/tcp, :::5432->5432/tcp   rengine_db_1

**Comment by blackpungas** [comment]
I had the same issue and try everything, couldnt solve it. i dont get why a 5k star repo have instalation issues.

**Comment by paweloque** [comment]
@blackpungas I think that being upset at the creators is not the best of ideas here. We better try to figure out why this is not working instead of complaining. I'm trying to execute collectstatic and serve the results via the django app. The proxy might have some problems do to domain configuration or certificates. I don't have a working solution yet though

**Comment by paweloque** [comment]
https://github.com/yogeshojha/rengine/issues/769#issuecomment-1365297325

**Comment by paweloque** [comment]
you can either set the DEBUG=1 parameter to serve static files via Django or change the `ssl_ecdh_curve` nginx param to
```
ssl_ecdh_curve                                  secp384r1:X25519:prime256v1;
```
And properly set your domain in the `.env` file. 
In my case this helped and I can properly start reNgine.

**Comment by gearcapitan** [comment]
> you can either set the DEBUG=1 parameter to serve static files via Django or change the `ssl_ecdh_curve` nginx param to
> 
> ```
> ssl_ecdh_curve                                  secp384r1:X25519:prime256v1;
> ```
> 
> And properly set your domain in the `.env` file. In my case this helped and I can properly start reNgine.

Didn't work for me!.

![imagen](https://user-images.githubusercontent.com/13740942/209615463-baad8286-098e-41d0-9689-fb41dbe60fd5.png)

**Comment by yogeshojha** [comment]
Hi @puimpsyc 

Please refer to the documentation: https://rengine.wiki/install/quick/#:~:text=reNgine%20can%20now%20be%20accessed%20from%20https%3A//127.0.0.1%20or%20if%20you%27re%20on%20the%20VPS%20https%3A//your_vps_ip_address

you are supposed to access in https://127.0.0.1 and not on port 8080

**Comment by yogeshojha** [comment]
Also, nginx issue has been fixed in v1.3.5


Thanks
